src/ui/standings.rs

```diff
@@ -56,7 +76,13 @@ impl StatefulWidget for StandingsWidget {
                     .border_type(BorderType::Rounded)
                     .padding(Padding::new(1, 1, 0, 0))
                     .title(Span::styled(
-                        state.date_selector.format_date_border_title(),
+                        format!("{} - {}", 
```

Comment:
would prefer to leave the title unchanged so it matches the other tabs

Reply:
👍